### ISSUE TITLE
Select boxes

### ISSUE THREAD

**Issue by raysguy** [raysguy/HTPC-Manager#4]
Thanks for the help @Hellowlol - adding the refresh line in makes it work :)

Only difficulty I'm having now is figuring out where and what to put in the js files that don't have the same format.  E.g in your example I could put in `$('#branch').selectpicker('refresh');` but in the couchpotato.js I can see functions relating to `<option>` but there is no # name for me to target and I don't know enough about js to figure it out!

Also is there an easier way to achieve the same result with some kind of global 'refresh' line or something at the end of each js file or does it need to be on each dropdown?

**Comment by Hellowlol** [comment]
I don't think this it's possible to have global one, or you can ..but you still need to call it after the HTML is made by the js. It's important that is no flickering, I hate that. I'll rather have ugly dropdowns.

You can add a id to the dropdowns or use something generic as $('select')....
